lib/reversetunnel/localsite.go

```diff
@@ -788,9 +802,9 @@ func (s *localSite) handleHeartbeat(rconn *remoteConn, ch ssh.Channel, reqC <-ch
 
 			log := logger
 			if roundtrip != 0 {
-				log = logger.WithField("latency", roundtrip.String())
+				log = logger.With("latency", logutils.StringerAttr(roundtrip))
```

Comment:
Sould we add special handling for `time.Duration` to our handlers so we don't have to do this every time, kinda like we do for `error`?